test/org/sagebionetworks/bridge/BridgeUtilsTest.java

```diff
@@ -52,24 +52,24 @@ public void periodsNotInterpretedAsRegex() {
     
     @Test
     public void commaListToSet() {
-        Set<String> set = BridgeUtils.commaListToSet("a, b , c");
-        assertEquals(Sets.newHashSet("a","b","c"), set);
+        Set<String> set = BridgeUtils.commaListToOrderedSet("a, b , c");
+        assertEquals(TestUtils.newLinkedHashSet("a","b","c"), set);
         
-        set = BridgeUtils.commaListToSet("a,b,c");
-        assertEquals(Sets.newHashSet("a","b","c"), set);
+        set = BridgeUtils.commaListToOrderedSet("a,b,c");
+        assertEquals(TestUtils.newLinkedHashSet("a","b","c"), set);
         
-        set = BridgeUtils.commaListToSet("");
-        assertEquals(Sets.newHashSet(), set);
+        set = BridgeUtils.commaListToOrderedSet("");
+        assertEquals(TestUtils.newLinkedHashSet(), set);
         
-        set = BridgeUtils.commaListToSet(null);
+        set = BridgeUtils.commaListToOrderedSet(null);
         assertNotNull(set);
         
-        set = BridgeUtils.commaListToSet(" a");
-        assertEquals(Sets.newHashSet("a"), set);
+        set = BridgeUtils.commaListToOrderedSet(" a");
+        assertEquals(TestUtils.newLinkedHashSet("a"), set);
         
         // Does not produce a null value.
-        set = BridgeUtils.commaListToSet("a,,b");
-        assertEquals(Sets.newHashSet("a","b"), set);
+        set = BridgeUtils.commaListToOrderedSet("a,,b");
+        assertEquals(TestUtils.newLinkedHashSet("a","b"), set);
```

Comment:
assertEquals(set1, set2) doesn't actually check the order of the sets are the same. Added a method to assert that as well, added additional test to change order of arguments.